tests/test_backends.py

```diff
@@ -1001,3 +1003,31 @@ def test_eigvalsh(self):
         result = gs.linalg.eigvalsh(mat, UPLO='U')
         expected = _np.linalg.eigvalsh(mat)
         self.assertAllCloseToNp(result, expected)
+
+    def test_cholesky(self):
+        mat = SPDMatrices(3).random_uniform(2)
+        result = gs.linalg.cholesky(mat)
+        expected = _np.linalg.cholesky(mat)
+        self.assertAllClose(result, expected)
+
+    @geomstats.tests.np_only
```

Comment:
this test has torch code, what doesn't work with pytorch?

Reply:
It does pass with torch but the goal was actually to use torch as reference and test numpy implementation here.